model/src/main/java/org/mskcc/cmo/metadb/model/SampleManifestJsonEntity.java

```diff
@@ -0,0 +1,46 @@
+package org.mskcc.cmo.metadb.model;
+
+import java.io.Serializable;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+
+@NodeEntity(label = "sample_manifest_json")
+public class SampleManifestJsonEntity implements Serializable {
+    @Id @GeneratedValue
+    private Long id;
+    private String sampleManifestJson;
+    private String creationDate;
+    
+    public SampleManifestJsonEntity() {}
+    
+    public SampleManifestJsonEntity(String json, String creationDate) {
+        this.sampleManifestJson = json;
+        this.creationDate = creationDate;
```

Comment:
can you also order the getters and setters in the same way that they are listed above? (id, sampleManifestJson, creationDate)